client/src/components/App/App.jsx

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+
+import Photo from '../Photo/Photo';
+import PhotoColumn from '../PhotoColumn/PhotoColumn';
+
+import 'normalize.css';
+import styles from './App.css';
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      photos: [],
+      loading: true,
+    };
+  }
+
+  componentDidMount() {
+    axios.get('/api/rooms/100/images')
+      .then(response => response.data)
+      .then((data) => {
+        this.setState({
+          photos: data,
+          loading: false,
+        });
+      })
+      // eslint-disable-next-line no-console
+      .catch(e => console.error(e));
+  }
+
+  render() {
+    const { photos, loading } = this.state;
+    const jsx = loading
```

Comment:
why not return instead of using jsx?

Reply:
@JayBo7 Good suggestion, I just made that change. Let me know what you think.